### PR TITLE
Chart UI updates for new metrics

### LINKED ISSUE #141: chromedash-featurelist on-scroll listener slows down initial load
When the features list first comes in, `scrollToMilestone` is called by chromedash-featurelist. Looking at a profile of the page load, this call takes ~100ms on my Z620. I suspect this is because of the `on-scroll` listener, which repeatedly tries to calculate which milestone to highlight in the left nav.

This could probably be avoided by waiting to add the on-scroll listener until after page load is finished.

### PR DESCRIPTION
**Pull request by ebidel**
R: @RByers 

Also adds a new trendline to the graphs.

screen shot 2017-11-01 at 10 35 18 am

Fixes #141

### CODE REVIEW

File: static/elements/chromedash-timeline.html [hunk 1]
```diff
@@ -56,21 +55,40 @@
...
       for (var i = 0, item; item = data[i]; ++i) {
         var dateStr = item.date.split('-');
         var date = new Date();
         date.setFullYear(parseInt(dateStr[0]), parseInt(dateStr[1]) - 1, parseInt(dateStr[2]));
-        rowArray.push([
-          date,
-          parseFloat((item.day_percentage * 100).toFixed(6))
-          // parseFloat(item.day_percentage * 100)
-        ]);
+        var row = [date, parseFloat((item.day_percentage * 100).toFixed(6))];
```
Comment (RByers) [inline static/elements/chromedash-timeline.html]: Is the `toFixed` here left over from the precision truncation stuff?  We don't need that anymore - the full precision exposed by the back-end handler is approved to be made public (see go/chromestatus-usecounter).

File: templates/metrics/css/animated.html [hunk 2]
```diff
@@ -30,7 +30,9 @@
   <div>
     <h3>About this data</h3>
     <p class="description">
-    We've been using Chrome's <a href="https://cs.chromium.org/chromium/src/tools/metrics/histograms/enums.xml" target="_blank">anonymous usage statistics</a> to count the CSS properties which are animated. A property that is animated multiple times during a single page view is counted each time it animates. <b>Percentages on this page indicate the amount of times (as the fraction of all animated properties) each property is used in a Transition or Animation</b>. Data is ~24 hrs old and reflects usage across all channels and platforms.</p>
+    We've been using Chrome's <a href="https://cs.chromium.org/chromium/src/tools/metrics/histograms/enums.xml" target="_blank">anonymous usage statistics</a> to count the CSS properties which are animated.
+    A property that is animated multiple times during a single page view is counted each time it animates.
```
Comment (RByers) [inline templates/metrics/css/animated.html]: The above line is no longer true, just delete it?  The statement below accurately reflects how it works.

File: static/elements/chromedash-timeline.html (hunk 1 above)
Comment (ebidel) [reply static/elements/chromedash-timeline.html]: Yes. Cool, I'll remove all the truncation stuff (backend and frontend parts).

Comment (ebidel) [reply static/elements/chromedash-timeline.html]: Actually, we may still need it for the historical data points. The raw metrics are saved and sent to the client truncated.

File: templates/metrics/css/animated.html (hunk 2 above)
Comment (ebidel) [reply templates/metrics/css/animated.html]: Do you mean just keep this sentence:

"Perrcentages on this page indicate ...

File: static/elements/chromedash-timeline.html (hunk 1 above)
Comment (RByers) [reply static/elements/chromedash-timeline.html]: Ugh. Maybe rely on some helper that does truncation just for data with old dates?

File: templates/metrics/css/animated.html (hunk 2 above)
Comment (RByers) [reply templates/metrics/css/animated.html]: Yes.  Remove line 34 (as it's contradictory to line 35 IMHO).

Comment (ebidel) [reply templates/metrics/css/animated.html]: done

File: static/elements/chromedash-timeline.html (hunk 1 above)
Comment (ebidel) [reply static/elements/chromedash-timeline.html]: Filed https://github.com/GoogleChrome/chromium-dashboard/issues/523 for now to track this. I don't think it hurts to keep the truncation in for now.

**Comment by ebidel** [comment]
PTAL

**Comment by maryroseMXXXVI** [comment]
Hi guys how it's going

**Pull request merged**